### ISSUE TITLE
Two horizontal scroll in version ^2.0.0-RC3

### ISSUE THREAD

**Issue by huynhsamha** [marmelab/react-admin#1819]
Hello, I'm using the react-admin version ^2.0.0-RC3, but I get an error on a table with multiple columns with 2 horizontal scroll (in body and table), the following image is my error:

![img](https://user-images.githubusercontent.com/24195776/39912234-499c5aee-5528-11e8-9254-8bd33b2cc6a5.png)

I want only horizontal scroll in table and columns is not stretch, so can you support with options in <Resource>

Thank you.

**Comment by fzaninotto** [comment]
I can reproduce it on Chrome mac with an extension to make the scrollbars visible. However I don't understand what could cause this...

**Comment by fzaninotto** [comment]
I suspect a bug cause by material-ui tooltips, because the (hidden) tooltips are the only element that go beyond the normal body width.

![image](https://user-images.githubusercontent.com/99944/39924795-820d823a-5529-11e8-91fe-ab3efaea6d1c.png)

**Comment by fzaninotto** [comment]
I think it's related to https://github.com/mui-org/material-ui/issues/9362. Do the second scrollbar appear right after loading the page, or only after you resize the browser window?

**Comment by huynhsamha** [comment]
Hello @fzaninotto, I'm using Chrome on Ubuntu.
First, only one scollbar of body on window. After I inspected elements on browser (that also resize the browser window), the second scrollbar appeared as my scnapshot (table has one and body also).

I'd like the toolbar with Add filter, Create and Refresh will be fixed on the top with the title of Resource. Current they is on the right of the table (when I load page or refresh, they can't be seen because it on the right).

**Comment by huynhsamha** [comment]
Hi @fzaninotto,

I current fixed my problem by add some css:
```scss
.layout {
    .list-page {
			table {
					width: 100%;
					overflow: auto;
            th,
            tr {
                span {
                    white-space: nowrap;
                    overflow: hidden;
                    text-overflow: ellipsis;
                }
            }
        }
    }
}

body {
		overflow-x: hidden;
		overflow-y: auto;
}
```

But I'd like the toolbar with Add filter, Create and Search, and Paging appeared fixed on screen, not scroll to right to view

**Comment by fzaninotto** [comment]
> When I don't inspect element, I also get the problem

Does it mean that it happens without resizing the browser window?

**Comment by huynhsamha** [comment]
Badly, I don't resize my browser, when i refresh page, the problem appeared.

**Comment by djhi** [comment]
Would you mind setting up a codesandbox ? You can use this one as a starting point: https://codesandbox.io/s/p3nq3lyjzq

**Comment by afilp** [comment]
As an enhancement, do you think that the paging buttons should be always **fixed** to the right, regardless of the horizontal scroll? 

![image](https://user-images.githubusercontent.com/7850073/40000506-a2ff654a-5794-11e8-9201-c69061a80f2d.png)

As it is now, you need to scroll horizontally 100% to be able to press the "Next" button.

**Comment by huynhsamha** [comment]
Hi @afilp 
Sorry you, maybe my bad english made you misunderstand my think. I want the toolbar and paging buttons should fixed on the right of the screen, not the body or layout, that mean when we open page, they should appear and we don't need scroll horizontally 100% to able to view or press them.

**Comment by afilp** [comment]
@huynhsamha Yes, this is what I also mean for the paging buttons, they should be visible always regardless of horizontal scroll.

**Comment by fzaninotto** [comment]
this can be achieved using a custom Pagination component, see https://marmelab.com/react-admin/List.html#pagination

**Comment by afilp** [comment]
@fzaninotto I assume your reply refers to the custom pagination only part, correct? I mean that the issue is still open for the double horizontal bars? This issue makes the UI quite awkward for non-tech users, as they have to click on both horizontal scrollbars to get to the end (horizontally).

**Comment by fzaninotto** [comment]
Yes, you're right. But I have trouble reproducing this issue. In my experience, it only happens if I resize the browser window after loading the admin.

Could you describe a way to reproduce it all the time?

**Comment by afilp** [comment]
I will do some tests and let you know. 

Just a note, I have this customization in `CardActions` so that the actions are not far away from the main screen, no need to scroll horizontally now:

```
const styles = {
    cardActions: {
        zIndex: 2,
        // display: 'flex',          
        // justifyContent: 'flex-end',
        // flexWrap: 'wrap',            
        position: 'absolute',           
        right: 0,                      
    },
};

const CardActions = ({ classes, className, children, ...rest }) => (
```

I doubt this is the reason, I just mention it.

**Comment by afilp** [comment]
@fzaninotto Please check this screenshot:

![image](https://user-images.githubusercontent.com/7850073/41547781-cfbe41c2-7329-11e8-9073-be2425340189.png)

There are other fields (columns) at the right side but they are hidden by `overflow: hidden`

There is a horizontal scroller, but we cannot see these columns:

![image](https://user-images.githubusercontent.com/7850073/41547912-2720fc98-732a-11e8-9cf0-f29cd17cca79.png)

**Comment by fzaninotto** [comment]
Fixed by #2206
